app/src/main/java/wei/mark/standouttest/MainActivity.java

```diff
@@ -0,0 +1,62 @@
+package wei.mark.standouttest;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Build;
+import android.provider.Settings;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import hyperionics.com.standouttest.R;
+import wei.mark.standouttest.floatingfolders.FloatingFolder;
+import wei.mark.standout.StandOutWindow;
+
+public class MainActivity extends AppCompatActivity {
+    public final static int REQUEST_CODE = 101;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
```

Comment:
do you need to set a content view when the activity isn't meant to display anything?